### DISCUSSION TITLE
What exactly happens when I reorder an entry within a filtered structure?

### DISCUSSION THREAD

**Discussion by outline4** [craftcms/cms#11185] [Q&A] [answered]
From what I've just tested the global order is destroyed and the element will be in a completely different order below the first entry with a different relation...

Can somebody elaborate what exactly happens?

**Comment by brandonkelly** [comment, marked as answer]
The entry will be repositioned in the section’s structure, relative to whatever element you are positioning it after. Or if no entries come before it, it will be moved to the top.

**Comment by outline4** [comment]
Thanks for the clarification. 

What happens with multi level structures? 

I've tried it and the results seem natural and correct, but I didn't figure out the logic behind it.

**Reply by brandonkelly** [reply]
Same answer as before.
